shared/src/price_estimate.rs

```diff
@@ -128,13 +132,25 @@ impl UniswapPriceEstimator {
             |_, path, pools| estimate_spot_price(path, pools),
         )
         .await
-        .and_then(|(path, price)| {
-            Ok((
-                path,
-                big_rational_to_float(price)
-                    .ok_or_else(|| anyhow!("Cannot convert price ratio to float"))?,
-            ))
-        })
+    }
+
+    /*
+        Returns a vector of (rational) prices for the given tokens
+        denominated in denominator_token.
+    */
```

Comment:
nanonit: can we keep the same format for docstrings as in the other public methods? That is just start each line with //